lib/travis/build/git/clone.rb

```diff
@@ -56,6 +72,14 @@ def quiet?
             config[:git][:quiet]
           end
 
+          def sparseCheckout
+            config[:git][:sparseCheckout]
+          end
+
+          def sparseCheckout?
+            !!config[:git][:sparseCheckout]
```

Comment:
Sorry that I'm not familiar with ruby, I'm not sure what should I do here, any tip please? thanks.